packages/util-body-length-node/src/calculateBodyLength.ts

```diff
@@ -0,0 +1,21 @@
+import { fstatSync, lstatSync } from "fs";
+
+export const calculateBodyLength = (body: any): number | undefined => {
+  if (!body) {
+    return 0;
+  }
+  if (typeof body === "string") {
+    return Buffer.from(body).length;
```

Comment:
If you're already changing this method, can I recommend changing this to `Buffer.byteLength(body)` instead? I think that you would have the same behaviour, just without creating a new buffer.

Also, I would be thrilled if you or another member of your team could take a look at my PR which has other performance improving changes: https://github.com/aws/aws-sdk-js-v3/pull/2612